.github/workflows/preflight.yml

```
@@ -0,0 +1,54 @@
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r requirements-dev.txt
+
+      - name: Mypy (mypy.ini scope)
+        run: mypy .
+
+      - name: Generate per-run Fernet key
+        # Throwaway key scoped to this CI run; production sets SESSION_COOKIE_KEY
+        # via Vercel env. Echoed to $GITHUB_ENV so subsequent steps see it.
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:123: Double quote to prevent globbing and word splitting

(shellcheck)